pkg/installer/generateconfig_test.go

```diff
@@ -45,3 +58,193 @@ func TestVMNetworkingType(t *testing.T) {
 		})
 	}
 }
+
+func TestApplyInstallConfigCustomisations(t *testing.T) {
```

Comment:
I think this needs to go in a separate file and be protected by an explicit build tag (like e.g. `// +build integration`), since this can't ever pass in CI.